reinforcement_learning/Lesson-7-MARL-draft.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>Typo: "with a" to "with an"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>